components/scheduler/views/overview.md

```diff
@@ -21,10 +21,11 @@ You can read more about this in the [Navigation]({%slug scheduler-navigation%})
 The available views are:
 
 * [Scheduler**Day**View]({%slug scheduler-views-day%})
-* [Scheduler**MultiDay**View]({%slug scheduler-views-multiday%})
 * [Scheduler**Week**View]({%slug scheduler-views-week%})
+* [Scheduler**MultiDay**View]({%slug scheduler-views-multiday%})
```

Comment:
Is this rendering right (note the lack of spaces)?

Scheduler**MultiDay**View

Reply:
I'm not sure here, I fully understand your question. The expected result should be Scheduler**MultiDay**View without spaces.